packages/grid/src/GridUtils.ts

```diff
@@ -85,21 +158,45 @@ class GridUtils {
     return undefined;
   }
 
-  static isInItem(itemIndex, itemXs, itemSizes, x) {
-    const itemX = itemXs.get(itemIndex);
-    const itemSize = itemSizes.get(itemIndex);
+  /**
+   * Check if the coordinate is within the item specified in this dimension
+   * @param itemIndex Index of the item to check
+   * @param itemXs Coordinate of all items in this dimension
+   * @param itemSizes Size of all items in this dimension
+   * @param x The coordinate to check
```

Comment:
Rename `itemXs` to `itemCoordinates` and `x` to `coordinate` would be more accurate I think